install_config/persistent_storage/dynamically_provisioning_pvs.adoc

```diff
@@ -113,6 +113,79 @@ value, triggers provisioning using the single implied
 link:#enabling-provisioner-plugins[provisioner plug-in per cloud].
 ====
 
+
+[[storage-provisioner-labels]]
+=== Storage-Provisioner Labels
+
+For dynamically provisioned storage, OpenShift defines three key/value pairs,
+called _storage-provisioner labels_ in this document,
+and associates them with the storage, in a storage-specific way.
+These are normally not visible to OpenShift users.
+Their main benefit is to enable storage administrators to recognize volumes
+associated with OpenShift.
+
+The term _storage-provisioner label_ is what this documentation uses.
+The actual term that a storage administrator will understand is listed
+in the following table:
```

Comment:
@bfallonf 
Right, "understand" is not the best choice.

Hmm, getting rid of *storage-provisioner label* entirely is a good idea.